act/discovery/__init__.py

```diff
@@ -4,8 +4,15 @@
 
 """
 
-from .get_armfiles import download_data
-from .get_asos import get_asos
-from .get_cropscape import croptype
-from .get_airnow import get_airnow_bounded_obs, get_airnow_obs, get_airnow_forecast
-from .get_noaapsl import download_noaa_psl_data
+import lazy_loader as lazy
+
+__getattr__, __dir__, __all__ = lazy.attach(
+    __name__,
+    submodules=['get_armfiles', 'get_cropscape'],
```

Comment:
Should we just have get_armfiles in here? Cropscape is not a high priority to load in initially

Reply:
We should have all them here - this lazily loads in the discovery submodules (`get_armfiles`, `get_cropscape`, etc.)